advanced/am-pattern-4/templates/am/gateway-external/wso2am-pattern-4-am-gateway-external-volume-claim.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@chathuraswso2 shall we please update the folder structure as follows?

```
gateway
├── external
└── internal
```

IMO, we do not have to repeat the term `gateway-` in every name and this structure looks clean.

Please feel free to drop in your thoughts and concerns.